libs/labelbox/src/labelbox/schema/create_batches_task.py

```diff
@@ -23,6 +23,9 @@ def __init__(self, client, project_id: str, batch_ids: List[str],
             Entity.Task.get_task(self.client, task_id) for task_id in task_ids
         ]
 
+    def wait_until_done(self, timeout_seconds: int = 300) -> None:
+        self.wait_till_done(timeout_seconds)
```

Comment:
Might consider writing a unit tests (e.g.) copy and paste to ensure if they are changed accidentally, nothing breaks